ci/README.md

```diff
@@ -0,0 +1,70 @@
+# VIP Go CI Scripts
+
+## `deploy.sh` and `deploy-exclude.txt`
+
+Scripts designed to facilitate building your Javascript and CSS
+on either [Travis CI](https://travis-ci.com) or [Circle CI](https://circleci.com/).
+
+See our documentation here: (NEEDS LINK, Ed)
+
+Sample Circle CI config, put into `.circleci/config.yml`:
```

Comment:
I think it would be helpful to point out the most important and relevant bits from the config files for repos that may already have have a config file set up (i.e. 1) branch ignore and 2) `steps`+`deploy` for Circle / `after_success` for Travis).

Reply:
👍 I've added more commentary in the examples